otaclient/app/boot_control/_common.py

```diff
@@ -415,6 +681,7 @@ def _mount_refroot(
         active_dev: str,
         standby_as_ref: bool,
     ):
+        CMDHelperFuncs.umount(self.ref_slot_mount_point)
```

Comment:
Where is the `ref_slot_mount_point` set in RPI case.
`ref_slot_mount_point` is not set in `otaclient/app/boot_control/_rpi_boot.py`.

Reply:
This method(and actually the whole `PrepareMountMixin`) is not used in rpi_boot implementation. Please check the above comment I left.

Reply:
I understand this PR strategy. But why is this change required even it is not used in `_rpi_boot.py`.

Reply:
updated in [4b324af](https://github.com/tier4/ota-client/pull/182/commits/4b324af60ee068c7de794e7935ed6bd6549ecbc1).